{{ cookiecutter.__package_name_kebab_case }}/pyproject.toml

```diff
@@ -143,6 +144,17 @@ source = ["src"]
 [tool.coverage.xml]
 output = "reports/coverage.xml"
 
+# https://flake8.pycqa.org/en/latest/user/options.html#options-and-their-descriptions
+# https://github.com/terrencepreilly/darglint#flake8
+[tool.flake8]
+color = "always"
+docstring_style = "{{ cookiecutter.docstring_style|lower }}"
+doctests = true
+ignore = ["DAR103", "E203", "E501", "W503"]
+max_line_length = 100
+max_complexity = 10
```

Comment:
How many years did they take to make this happen? :p That's great news :)

Reply:
Indeed! Using a flake8 extension is still somewhat of a workaround though, I hope flake8 will come around and support pyproject.toml in the future.